cli/tests/integration_tests.rs

```diff
@@ -2062,6 +2062,18 @@ fn test_permissions_rw_no_prefix() {
   }
 }
 
+#[test]
+fn test_permissions_read_watch_fs_with_relative_path() {
+  let (_, err) = util::run_and_collect_output(
+    true,
+    "run --allow-read=. complex_permissions_test.ts readWatchFs tls",
```

Comment:
Shouldn't it be `./tls` if you are testing for relative path?

Reply:
Yea sir